components/admin_console/system_users/system_users.jsx

```diff
@@ -304,7 +304,7 @@ export default class SystemUsers extends React.Component {
                         value={this.props.filter}
                         onChange={this.handleFilterChange}
                     >
-                        <option value=''>{'None'}</option>
+                        <option value=''>{'All Users'}</option>
```

Comment:
I'm not sure why the original "None" string wasn't translatable, but this should use `Utils.localizeMessage` like the other options so that it can be translated into other languages. You'll also need to add a corresponding entry to `i18n/en.json` with the matching English string.

Reply:
Echoing @hmhealey on this one. Do we also need a ticket to add translations for this string?